### ISSUE TITLE
Multiple pickers in one activity

### ISSUE THREAD

**Issue by jerome2710** [jkwiecien/EasyImage#198]
Would it be possible to use multiple pickers in a single activity? Now, both pickers will return in `onActivityResult`, but how could I pass a flag to the builder to recognise which picker is returning a result?

**Comment by jkwiecien** [comment]
use multiple EasyImage instances

**Comment by jerome2710** [comment]
Such a useless and obvious reply.. Of course, multiple EasyImage instances need to be used. Let me try again...

Two onClickListeners are created:

```java
findViewById(R.id.view_a).setOnClickListener(new View.OnClickListener() {
    @Override
    public void onClick(View view) {
        mImagePickerA = new EasyImage.Builder(ActivityClass.this).build();
        mImagePickerA.openChooser(ActivityClass.this);
    }
});

findViewById(R.id.view_b).setOnClickListener(new View.OnClickListener() {
    @Override
    public void onClick(View view) {
        mImagePickerB = new EasyImage.Builder(ActivityClass.this).build();
        mImagePickerB.openChooser(ActivityClass.this);
    }
});
```

When onActivityResult gets called, how do I recognize which instance is returning a result?

```java
@Override
protected void onActivityResult(int requestCode, int resultCode, Intent data) {
    mImageA.handleActivityResult(requestCode, resultCode, data, this, new DefaultCallback() {
        @Override
        public void onMediaFilesPicked(@NotNull MediaFile[] mediaFiles, @NotNull MediaSource mediaSource) {
            ...
        }
    });

    mImageB.handleActivityResult(requestCode, resultCode, data, this, new DefaultCallback() {
        @Override
        public void onMediaFilesPicked(@NotNull MediaFile[] mediaFiles, @NotNull MediaSource mediaSource) {
            ...
        }
    });
}
```

**Comment by jkwiecien** [comment]
either `mImageA` or `mImageB`  `onMediaFilesPicked` gets called?
